agent/src/Ingest.php

```diff
@@ -39,8 +39,8 @@ class Ingest
      * @param  LoopInterface  $loop
      * @param  Browser  $browser
      * @param  (Closure(ResponseInterface $response, float $duration): mixed)  $onIngestSuccess
-     * @param  (Closure(Throwable $e, float $duration): mixed)  $onIngestError
-     * @param  (Closure(float $duration): mixed)  $onOverQuota
+     * @param  (Closure(string $message, float $duration): mixed)  $onIngestError
+     * @param  (Closure(string $message, float $duration): mixed)  $onOverQuota
```

Comment:
Did some standardisation of making these receive strings rather than exceptions.